### PR TITLE
Prev6

### PR DESCRIPTION
**Pull request by morenol**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -14,17 +14,17 @@ jobs:
...
+            "functions/flat-map",
+            "functions/map",
+            "functions/ref-state",
+            "functions/update-state",
+            "use-cases/bank-processing",
+            "use-cases/http-callout",
+            "use-cases/word-counter",
           ]
-          ssdk-version: [ssdk-preview5, ssdk-preview6-dev]
+          ssdk-version: [ssdk-preview7-dev]
```
Comment (morenol) [inline .github/workflows/ci.yml:27]:
```suggestion
          ssdk-version: [ssdk-preview6, ssdk-preview7-dev]
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -14,17 +14,17 @@ jobs:
...
+            "functions/flat-map",
+            "functions/map",
+            "functions/ref-state",
+            "functions/update-state",
+            "use-cases/bank-processing",
+            "use-cases/http-callout",
+            "use-cases/word-counter",
           ]
-          ssdk-version: [ssdk-preview5, ssdk-preview6-dev]
+          ssdk-version: [ssdk-preview6, ssdk-preview7-dev]
```
Comment (sehz) [inline .github/workflows/ci.yml:27]: let's remove `ssdk-preview7-dev` as this should only work on stable

Comment (morenol) [inline .github/workflows/ci.yml:27]:
```suggestion
          ssdk-version: [ssdk-preview6]
```

**Comment by sehz** [comment]
Also, can you add `done`  so we can enable `merge-queue` ?

File: .github/workflows/ci.yml (hunk 2 above)
Comment (morenol) [reply .github/workflows/ci.yml]: done

**Comment by morenol** [comment]
> Also, can you add `done` so we can enable `merge-queue` ?

Acting that using `Done` is not a good approach in merge_queues right now because I think that merge_queues does not behave as expected when the job is skipped (Done job is skipped when any of their dependencies failed). 

Instead I think that we should add explicit references to the checks needed to merge

**Comment by sehz** [comment]
Sounds good

**Review by sehz — APPROVED** [review]
LGTM

**Comment by sehz** [comment]
in that case, instead `jobs` -> `validate` ?

**Pull request closed without merging**